docs/source/cli.rst

```diff
@@ -20,7 +20,10 @@ Discovery
 *********
 
 The tool can automatically discover supported devices using a broadcast-based discovery protocol.
-This works by sending an UDP datagram on port 9999 to the broadcast address (defaulting to ``255.255.255.255``).
+This works by sending an UDP datagram on ports 9999 and 20002 to the broadcast address (defaulting to ``255.255.255.255``).
+
+Newer devices that respond on port 20002 will most likely require TP-Link cloud credentials to be passed or they will report as having failed authentication.
+Use ``--username`` and ``--password`` options to specify credentials.  These values can also be set as environment variables via KASA_USERNAME and KASA_PASSWORD.
```

Comment:
```suggestion
Use ``--username`` and ``--password`` options to specify credentials.
These values can also be set as environment variables via ``KASA_USERNAME`` and ``KASA_PASSWORD``.
```

Reply:
Aargh I can't believe I did that.  Now fixed.